students/cjfortu/lesson07/html_render.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+    indent = 4
+    tag = 'html'
+    kwargs = {}
```

Comment:
this should not be a class variable  - class variables values don't change between instances but in reality your kwargs would change on each instance created.